CONFIGURATION.md

```diff
@@ -113,21 +127,48 @@ strains:
       /pipeline: https://github.com/adobe/helix-pipeline.git
       /cli: https://github.com/adobe/helix-cli.git
 ```
+</del>
 
 ## Proposal: `mount` points can take Git URLs, `content` objects/references, Backend URLs, `origin` objects/references
 
+> Notes:
+> We decided during the 4/2018 hackathon not to introduce mount points, so that it will be X strains for X mount points.
+
+<del>
 That way you can have a consistent traffic mapping without having to switch strains.
+</del>
 
 ## Proposal: `hlx deploy` lists all strains that are affected by the deployment and suggests the creation of a new strain if none are affected
 
+> Notes:
+> - `hlx deploy` always acts on only 1 strain. which is either the name of the current branch, dirty or what is specified with `--strain`
```

Comment:
This is not my understanding. `hlx deploy` creates multiple actions, and the full path of the action (package + name) is dependent on:
- file name, e.g. `html.htl` becomes `/*/html`
- current sha, e.g. `b894d631b417a1367a0c2be8ec8ed7688d176e70` becomes `/b894d631b417a1367a0c2be8ec8ed7688d176e70/*`
- `--dirty` flag, e.g. `/b894d631b417a1367a0c2be8ec8ed7688d176e70-dirty/*`

This deployment package can then be referenced from multiple strains, but at the deploy time, `hlx` does not need to have any knowledge of strains. In contrast, `hlx publish` needs to have knowledge of deployment packages, and we need to pass this knowledge on to the user or update the configuration, if needed.

Reply:
yes. you are correct. deploy doesn't care about strains.